From c02535e82a1a6e7bd156b08f2a7e8a35b64c48c5 Mon Sep 17 00:00:00 2001
From: jollaman999 <admin@jollaman999.com>
Date: Tue, 5 Jun 2018 11:10:32 +0900
Subject: [PATCH 2/5] ar71xx: switch to 4.14 kernel

Signed-off-by: jollaman999 <admin@jollaman999.com>
---
 target/linux/ar71xx/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/linux/ar71xx/Makefile b/target/linux/ar71xx/Makefile
index 104961929f..71fe02f326 100644
--- a/target/linux/ar71xx/Makefile
+++ b/target/linux/ar71xx/Makefile
@@ -13,7 +13,7 @@ FEATURES:=usbgadget
 CPU_TYPE:=24kc
 SUBTARGETS:=generic tiny nand mikrotik
 
-KERNEL_PATCHVER:=4.9
+KERNEL_PATCHVER:=4.14
 
 include $(INCLUDE_DIR)/target.mk
 
-- 
2.17.1

